FourthTask/figures/tests/tests.go

```diff
@@ -0,0 +1,51 @@
+//	Tests package. Here i'll write some tests and if smth will go wrong i'll tell main pocket to stop his work
+package tests
+
+import (
+	"errors"
+	"math"
+
+	"github.com/Mishamba/EPAM_GOLANG_COURSE/FourthTask/figures/types"
+)
+
+//i don't know, what else to test here
+
+func Errors() error {
+	_, err := types.CircleConstruct(-5) //negative input
+	if err != nil {
+		return err
```

Comment:
Possibly there must be an error. We can't create a circle with negative radius, so this func should return an error, and test should fail if no error was returned, like 
```go
if err := types.CircleConstruct(-5); err == nil {
   panic(errors.New("we are from outer space")
}
```